suite/suite.go

```diff
@@ -82,15 +82,8 @@ func Run(t *testing.T, suite TestingSuite) {
 	suite.SetT(t)
 	defer failOnPanic(t)
 
-	if setupAllSuite, ok := suite.(SetupAllSuite); ok {
-		setupAllSuite.SetupSuite()
-	}
-	defer func() {
-		if tearDownAllSuite, ok := suite.(TearDownAllSuite); ok {
-			tearDownAllSuite.TearDownSuite()
-		}
-	}()
-
+	setupDone := false
```

Comment:
Can we rename this to something like `suiteSetupDone`?

Reply:
Sure!